checkstyle/checkstyle.xml

```diff
@@ -6,7 +6,7 @@
 <module name="Checker">
   <module name="TreeWalker">
     <module name="ImportControl">
-      <property name="file" value="checkstyle/import_control.xml"/>
```

Comment:
Did you mean to leave this changed, or was it part of trying to get the checkstyle failure to work?

Reply:
@jliunyu, that's really weird - what kind of errors did you get before and after this change and with what Maven command? Can you please also check whether non-test targets (e.g. `clean package` or `validate`) are behaving as expected before and after the changes?

Reply:
This one is just for removing the checkstyle failure, I'm will remove this change.